util/importers/LDC97S62.py

```diff
@@ -1,39 +1,35 @@
-from __future__ import print_function
-from __future__ import absolute_import
+from __future__ import absolute_import, division, print_function
+
 import fnmatch
-import numpy as np
 import os
+import pandas
 import subprocess
-import wave
 import tensorflow as tf
 import unicodedata
-import codecs
-from glob import glob
-from itertools import cycle
+import wave
+
 from math import ceil
-from Queue import PriorityQueue
-from Queue import Queue
+from six.moves import range
```

Comment:
Do we really need to import `range` here or can we simply rely on the python 2.x or python 3.x `range`?

We only use `range` here
```python
batch_threads = [Thread(target=self._populate_batch_queue, args=(session,)) for i in range(self._thread_count)]
```
and `self._thread_count` isn't large.  So I don't see any harm in simply falling back to python 2.x or python 3.x `range`.

Is there some problem with my logic?